packages/react/src/primitives/Link/__tests__/Link.test.tsx

```diff
@@ -87,7 +87,7 @@ describe('Link: ', () => {
     expect(link.nodeName).toBe('P');
   });
 
-  it('can apply styling via props', async () => {
+  it.only('can apply styling via props', async () => {
```

Comment:
Oh shoot, I should remove `only`. Will update later

Reply:
😈